pkg/pf/tfgen/testdata/TestWriteOnlyOmit.golden

```diff
@@ -28,5 +32,33 @@
                 "type": "object"
             }
         }
+    },
+    "functions": {
```

Comment:
Good opportunity to make these tests less pesky by not overfitting on keys they do not care about. For example using https://github.com/pulumi/providertest/commit/e3233fc0cba08053190b35d196479057d45f7b17

Or with some work converting them to autogold tests so we can take a PR and then auto-update all gold tests with one command.